src/wp-content/themes/twentytwentyone/classes/class-twenty-twenty-one-dark-mode.php

```diff
@@ -18,6 +18,38 @@ class Twenty_Twenty_One_Dark_Mode {
 	 * @since Twenty Twenty-One 1.0
 	 */
 	public function __construct() {
+		if ( is_admin() ) {
+			$this->setup_admin();
+		} else {
+			$this->add_hooks();
+		}
+	}
+
+	/**
+	 * Sets up Dark Mode for use within the admin area.
+	 *
+	 * @since Twenty Twenty-One 1.4
+	 */
+	public function setup_admin() {
+		add_action( 'current_screen', array( $this, 'add_hooks' ) );
+
+		// Add the privacy policy content. This is done here because current_screen is too late.
+		add_action( 'admin_init', array( $this, 'add_privacy_policy_content' ) );
+
+		// Add customizer controls.
+		add_action( 'customize_register', array( $this, 'customizer_controls' ) );
+	}
+
+	/**
+	 * Adds the needed hooks to support Dark Mode.
+	 *
+	 * @since Twenty Twenty-One 1.4
+	 */
+	public function add_hooks() {
+		// Disable Dark Mode in the widgets editor.
+		if ( ( is_admin() && 'widgets' === get_current_screen()->base ) || is_customize_preview() ) {
```

Comment:
`is_customize_preview` here breaks the customizer preview... The preview should continue to work with dark-mode, the problem was in the sidebar when going in the "widgets" section. I see however that this has already been fixed and is no longer an issue...
To provide some context, if I remove the `is_customize_preview()` condition from this line I still see the issue when I enable Gutenberg as a plugin: 
![Screenshot from 2021-07-09 11-33-58](https://user-images.githubusercontent.com/588688/125049409-9e02a380-e0a9-11eb-94e1-c371171e4970.png)
The preview itself is fine and should not change, but the widgets is the sidebar are not. That's where the issue was... This seems to be OK on wp-trunk now without Gutenberg enabled (maybe some other commit recent commit fixed it)